web/utils.py

```
@@ -942,3 +942,28 @@
     except Exception as e:
         logger.error(f"Geocoding error: {e}")
         return None
+
+
+def get_email_status(profile):
```

Ruff (0.8.2)

Missing return type annotation for public function `get_email_status`

(ANN201)